sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -222,33 +224,105 @@ def test_random_starts():
 
 @pytest.mark.parametrize('kernel', kernels)
 def test_y_normalization(kernel):
-    # Test normalization of the target values in GP
+    """
+    Test normalization of the target values in GP
 
-    # Fitting non-normalizing GP on normalized y and fitting normalizing GP
-    # on unnormalized y should yield identical results
-    y_mean = y.mean(0)
-    y_norm = y - y_mean
+    Fitting non-normalizing GP on normalized y and fitting normalizing GP
+    on unnormalized y should yield identical results. Note that, here,
+    'normalized y' refers to y that has been made zero mean and unit
+    variance.
+
+    """
+
+    y_mean = np.mean(y)
+    y_std = np.std(y)
+    y_norm = (y - y_mean) / y_std
 
     # Fit non-normalizing GP on normalized y
     gpr = GaussianProcessRegressor(kernel=kernel)
     gpr.fit(X, y_norm)
+
     # Fit normalizing GP on unnormalized y
     gpr_norm = GaussianProcessRegressor(kernel=kernel, normalize_y=True)
     gpr_norm.fit(X, y)
 
     # Compare predicted mean, std-devs and covariances
     y_pred, y_pred_std = gpr.predict(X2, return_std=True)
-    y_pred = y_mean + y_pred
+    y_pred = y_pred * y_std + y_mean
+    y_pred_std = y_pred_std * y_std
     y_pred_norm, y_pred_std_norm = gpr_norm.predict(X2, return_std=True)
 
     assert_almost_equal(y_pred, y_pred_norm)
     assert_almost_equal(y_pred_std, y_pred_std_norm)
 
     _, y_cov = gpr.predict(X2, return_cov=True)
+    y_cov = y_cov * y_std**2
     _, y_cov_norm = gpr_norm.predict(X2, return_cov=True)
+
     assert_almost_equal(y_cov, y_cov_norm)
 
 
+def test_large_variance_y():
+    """
+    Here we test that, when noramlize_y=True, our GP can produce a
+    sensible fit to training data whose variance is significantly
+    larger than unity. This test was made in response to issue #15612.
+
+
+    GP predictions are verified against predictions that were made
+    using GPy which, here, is treated as the 'gold standard'. Note that we
+    only investigate the RBF kernel here, as that is what was used in the
+    GPy implementation.
+
```

Comment:
rm blank line

Reply:
Done. Also corrected a small typo in the test description.